src/couch_replicator_scheduler.erl

```diff
@@ -284,13 +296,12 @@ running_job_count() ->
 
 -spec running_jobs() -> [#job{}].
 running_jobs() ->
-    ets:tab2list(?MODULE) -- pending_jobs().
+    ets:select(?MODULE, [{#job{pid = '$1', _='_'}, [{is_pid, '$1'}], ['$_']}]).
```

Comment:
nice. can we pull this into a separate commit, though? (since it's not about adding exponential backoff)